wagtail/wagtailadmin/templates/wagtailadmin/pages/listing/_page_title_choose.html

```diff
@@ -5,7 +5,7 @@
 {% endcomment %}
 
 <h2>
-    {% if page.can_choose %}
+    {% if page.can_choose and not page.is_root %}
```

Comment:
Yep - looks like there ought to be a `and not parent_page.is_root()` check there.